File: common/src/main/java/com/concessions/common/network/NetworkConstants.java
package com.concessions.common.network;

public interface NetworkConstants {
	
	// Configuration service
	public String CONFIGURATION_SERVICE = "CONFIG";
	public String CONFIGURATION_LOCATION_ACTION = "LOCATION";
	
	// Device service
	public String DEVICE_SERVICE = "DEVICE";
	public String DEVICE_REGISTER_ACTION = "REGISTER";
	
	// Health Check Service
	public String HEALTH_SERVICE = "HEALTH";
	public String HEALTH_CHECK_ACTION = "CHECK";

	// Journal Service
	public String JOURNAL_SERVICE = "JOURNAL";
	public String JOURNAL_GET_ACTION = "GET";
	public String JOURNAL_CHANGE_ACTION = "CHANGE";

	// Menu service
	public String MENU_SERVICE = "MENU";
	public String MENU_GET_ACTION = "GET";
	
	// Order Service
	public String ORDER_SERVICE = "ORDER";
	public String ORDER_SUBMIT_ACTION = "SUBMIT";
	
	// PIN Service
	public String PIN_SERVICE = "PIN";
	public String PIN_VERIFY_ACTION = "VERIFY";
}

File: app/lib/network/network_constants.dart
class NetworkConstants {
  // To prevent the class from being instantiated
  NetworkConstants._();

  // Configuration service
  static const String configurationService = "CONFIG";
  static const String configurationLocationAction = "LOCATION";

  // Device service
  static const String deviceService = "DEVICE";
  static const String deviceRegisterAction = "REGISTER";

  // Health Check Service
  static const String healthService = "HEALTH";
  static const String healthCheckAction = "CHECK";

  // Journal Service
  static const String journalService = "JOURNAL";
  static const String journalGetAction = "GET";
  static const String journalChangeAction = "CHANGE";

  // Menu service
  static const String menuService = "MENU";
  static const String menuGetAction = "GET";

  // Order Service
  static const String orderService = "ORDER";
  static const String orderCompleteAction = "COMPLETE";
  static const String orderCompletedAction = "COMPLETED";
  static const String orderCreatedAction = "CREATED";
  static const String orderGetallAction = "GETALL";
  static const String orderSubmitAction = "SUBMIT";

  // PIN Service
  static const String pinService = "PIN";
  static const String pinVerifyAction = "VERIFY";
}